lib/App.server.js

```diff
@@ -262,9 +262,13 @@ App.prototype._watchViews = function(filenames, filename, namespace) {
   var watcher = chokidar.watch(filenames);
   watcher.on('change', function() {
     watcher.close();
-    app.loadViews(filename, namespace);
-    app._updateScriptViews();
-    app._refreshClients();
+    try {
```

Comment:
Good call. The error's in `_updateScriptViews` so we can be reliably sure that the watcher gets restored.

I narrowed the try/catch blocks to wrap only that line